src/biome/text/tokenizer.py

```diff
@@ -227,16 +225,3 @@ def _tokenize(self, text: str) -> List[Token]:
     @property
     def nlp(self) -> Language:
         raise NotImplementedError("For the TransformerTokenizer we have no spaCy nlp")
-
-
-def _fetch_spacy_model(lang: str):
-    # Allennlp get_spacy_model method works only for fully named models (en_core_web_sm) but no
-    # for already linked named (en, es)
-    # This is a workaround for mitigate those kind of errors. Just loading one more time, it's ok.
-    # See https://github.com/allenai/allennlp/issues/4201
-    import spacy
-
-    try:
-        spacy.load(lang, disable=["vectors", "textcat", "tagger" "parser" "ner"])
-    except OSError:
-        spacy.cli.download(lang)
```

Comment:
we no longer need download the model if doesn't exist?

Reply:
AllenNLP does this for you with its [get_spacy_model](https://github.com/allenai/allennlp/blob/cf113d705b9054d329c67cf9bb29cbc3f191015d/allennlp/common/util.py#L258)